arelle/plugin/validate/EDINET/rules/edinet.py

```diff
@@ -224,3 +225,27 @@ def rule_EC8062W(
                 assetSum=f"{assetSum:,}",
                 modelObject=facts,
             )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_EC8075W(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC8075W: The percentage of female executives has not been tagged in detail. Ensure that there is
+    a nonnil value disclosed for jpcrp_cor:RatioOfFemaleDirectorsAndOtherOfficers.
+    """
+    formTypes = pluginData.getDocumentTypes(val.modelXbrl)
+    if any(form in CORPORATE_DISCLOSURE_FORMS for form in formTypes):
+        requiredFact = val.modelXbrl.factsByQname.get(pluginData.ratioOfFemaleDirectorsAndOtherOfficersQn, set())
+        if not len([fact for fact in requiredFact if fact is not None and fact.xValid >= VALID]):
```

Comment:
We seem to be repeating this a lot. Probably a good opportunity for a reusable `hasRequiredValidFact` or similar.